### PR TITLE
Nuget upgrades

### PR DESCRIPTION
**Pull request by JamieMagee**
#### Database Migration
NO

#### Description
Upgraded Nuget packages with updates available. Each commit represents a different dependency.

### CODE REVIEW

File: src/NzbDrone.Integration.Test/NzbDrone.Integration.Test.csproj [hunk 1]
```diff
@@ -38,20 +38,18 @@
...
+    <Reference Include="FluentAssertions.Core, Version=4.19.0.0, Culture=neutral, PublicKeyToken=33f2691a05b67b6a, processorArchitecture=MSIL">
+      <HintPath>..\packages\FluentAssertions.4.19.0\lib\net40\FluentAssertions.Core.dll</HintPath>
     </Reference>
     <Reference Include="FluentValidation, Version=6.2.1.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\FluentValidation.6.2.1.0\lib\portable-net40+sl50+wp80+win8+wpa81\FluentValidation.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.AspNet.SignalR.Client">
-      <HintPath>..\packages\Microsoft.AspNet.SignalR.Client.1.2.1\lib\net40\Microsoft.AspNet.SignalR.Client.dll</HintPath>
+    <Reference Include="Microsoft.AspNet.SignalR.Client, Version=2.2.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (JamieMagee) [inline src/NzbDrone.Integration.Test/NzbDrone.Integration.Test.csproj]: Okay, I've reverted the upgrade to 2.x, but still upgraded it from 1.2.1 to 1.2.2.

**Comment by JamieMagee** [comment]
I don't have access to an ubuntu host currently to test, but it looks to be related to the Moq upgrade, so I've reverted this for now.

**Comment by markus101** [comment]
Looks like the revert wasn't successful, tests are failing under Windows with:

```
ystem.IO.FileLoadException : Could not load file or assembly 'Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
  ----> System.IO.FileLoadException : Could not load file or assembly 'Moq, Version=4.0.10827.0, Culture=neutral, PublicKeyToken=69f491c39445e920' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
TearDown : System.IO.FileLoadException : Could not load file or assembly 'Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
  ----> System.IO.FileLoadException : Could not load file or assembly 'Moq, Version=4.0.10827.0, Culture=neutral, PublicKeyToken=69f491c39445e920' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at NzbDrone.Common.Test.Http.HttpClientFixture`1.SetUp()
--FileLoadException

--TearDown
   at NzbDrone.Test.Common.TestBase.TestBaseTearDown()
--FileLoadException
```

**Comment by JamieMagee** [comment]
I removed the last instance of the Moq upgrade. I'm not sure why Visual studio wasn't able to do it for me, but it worked manually changing it.

**Comment by markus101** [comment]
Thanks, I cleaned up the commits a bit and merged.

**Pull request closed without merging**